stdparams.go

```diff
@@ -0,0 +1,36 @@
+// +build go1.7
+
+package httprouter
+
+import (
+	"context"
+	"net/http"
+)
+
+// ParamsKey is the request context key under which URL params are stored.
+//
+// This is only present from go 1.7.
+const ParamsKey = "_httprouter_params"
```

Comment:
You may want to consider using an unexported type and key for this so that you're guaranteed to avoid collisions with other packages. See the userip example on the context blog post: https://blog.golang.org/context#TOC_3.2.